pages/linux/systemd-analyze.md

```diff
@@ -9,3 +9,11 @@
 - Print a tree of the time critical chain of units:
 
 `systemd-analyze critical-chain`
+
+- Create a SVG file showing when each system service started, highlighting the time they spent on initialization.
+
+`systemd-analyze plot > bootup.svg`
+
+- Plot a dependency graph and convert it into a SVG file
+
+`systemd-analyze dot | dot -Tsvg > dependencies.svg`
```

Comment:
```{{dependencies.svg}}```
@bl-ue ?

Reply:
Yeah you're right @Waples.  @Sematre, we usually wrap variables like this in our custom token syntax, like so:
```suggestion
`systemd-analyze dot | dot -T{{svg}} > {{dependencies.svg}}`
```